scrapers/govScrapers/getColombia.js

```diff
@@ -0,0 +1,53 @@
+const axios = require('axios');
+const logger = require('../../utils/logger');
+
+const filters = ['Recuperado', 'Fallecido'];
```

Comment:
Why do you have this variable in an Array? Just use the strings as keys on lines 11&12